src/request_collector/server.py

```diff
@@ -24,26 +44,77 @@ def error_handler(context, exc_info):
     sys.exit()
 
 
+def setup_matrix(private_key: str, chain_id: int) -> Tuple[GMatrixClient, Room]:
+    available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[Environment.DEVELOPMENT]
+    available_servers = get_matrix_servers(available_servers_url)
+
+    def _http_retry_delay() -> Iterable[float]:
+        # below constants are defined in raiden.app.App.DEFAULT_CONFIG
+        return udp_utils.timeout_exponential_backoff(
+            DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
+            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL / 5,
+            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL,
+        )
+
+    client = make_client(
+        servers=available_servers,
+        http_pool_maxsize=4,
+        http_retry_timeout=40,
+        http_retry_delay=_http_retry_delay,
+    )
+
+    try:
+        login_or_register(client, signer=LocalSigner(private_key=decode_hex(private_key)))
+    except MatrixRequestError:
+        raise ConnectionError('Could not login/register to matrix.')
+    except ValueError:
+        raise ConnectionError('Could not login/register to matrix.')
+
+    try:
+        room_name = '_'.join(['raiden', str(chain_id), MONITORING_BROADCASTING_ROOM])
```

Comment:
https://github.com/raiden-network/raiden/pull/3471 is merged. Please, use that instead of this, as it also takes care of network name logic and so
```suggestion
        room_name = make_room_alias(chain_id, MONITORING_BROADCASTING_ROOM)
```